examples/widgets/search_entry.nim

```diff
@@ -0,0 +1,88 @@
+# MIT License
+# 
+# Copyright (c) 2023 Can Joshua Lehmann
+# 
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+# 
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+# 
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE
+
+import owlkettle, owlkettle/[playground, adw, dataentries]
+import std/[strutils, sequtils]
+
+viewable App:
+  searchDelay: uint = 100
+  text: string = ""
+  placeholderText: string = "Search List"
+  sensitive: bool = true
+  tooltip: string = ""
+  sizeRequest: tuple[x, y: int] = (-1, -1)
+  
+  items: seq[string] = mapIt(0..<100, "Search Entry " & $it)
+  filteredItems: seq[string] = mapIt(0..<100, "Search Entry " & $it)
```

Comment:
```suggestion
  items: seq[string] = mapIt(0..<100, "Item " & $it)
  filteredItems: seq[string] = mapIt(0..<100, "Item " & $it)
```

Otherwise `Search Entry` refers to two different things (1. a search result 2. the `SearchEntry`).